docker/README.md

```diff
@@ -61,44 +61,44 @@ Start `btcd`, and then create an address for `Alice` that we'll directly mine
 bitcoin into.
 ```shell
 # Init bitcoin network env variable:
-⛰  export NETWORK="simnet" 
+export NETWORK="simnet" 
 
 # Create persistent volumes for alice and bob.
-⛰  docker volume create simnet_lnd_alice
-⛰  docker volume create simnet_lnd_bob
+docker volume create simnet_lnd_alice
+docker volume create simnet_lnd_bob
 
 # Run the "Alice" container and log into it:
-⛰  docker-compose run -d --name alice --volume simnet_lnd_alice:/root/.lnd lnd
-⛰  docker exec -i -t alice bash
+docker-compose run -d --name alice --volume simnet_lnd_alice:/root/.lnd lnd
+docker exec -i -t alice bash
 
 # Generate a new backward compatible nested p2sh address for Alice:
-alice ⛰  lncli --network=simnet newaddress np2wkh 
+alice$ lncli --network=simnet newaddress np2wkh 
 
 # Recreate "btcd" node and set Alice's address as mining address:
-⛰  MINING_ADDRESS=<alice_address> docker-compose up -d btcd
+MINING_ADDRESS=<alice_address> docker-compose up -d btcd
 
 # Generate 400 blocks (we need at least "100 >=" blocks because of coinbase 
 # block maturity and "300 ~=" in order to activate segwit):
-⛰  docker exec -it btcd /start-btcctl.sh generate 400
+docker exec -it btcd /start-btcctl.sh generate 400
 
 # Check that segwit is active:
-⛰  docker exec -it btcd /start-btcctl.sh getblockchaininfo | grep -A 1 segwit
+docker exec -it btcd /start-btcctl.sh getblockchaininfo | grep -A 1 segwit
 ```
 
 Check `Alice` balance:
 ```shell
-alice ⛰  lncli --network=simnet walletbalance
+alice$ lncli --network=simnet walletbalance
```

Comment:
Don't these lines have the same problem as before? Maybe we should just delete the `alice$`